cmd/frontend/internal/cli/serve_cmd.go

```diff
@@ -126,12 +126,12 @@ func Main(enterpriseSetupHook func(db database.DB, c conftypes.UnifiedWatchable)
 	log.SetFlags(0)
 	log.SetPrefix("")
 
-	syncLogs := sglog.Init(sglog.Resource{
+	cb := sglog.InitWithSinks(sglog.Resource{
```

Comment:
```suggestion
	liblog := sglog.InitWithSinks(sglog.Resource{
```

https://github.com/sourcegraph/sourcegraph/pull/36569#discussion_r891417831